.coderabbit.yaml

```
@@ -0,0 +1,97 @@
+
+reviews:
+  profile: assertive
+  high_level_summary: true
+  review_status: true
+  collapse_walkthrough: false
+  poem: false
+
+  auto_review:
+    enabled: true
+    drafts: false
+#    ignore_title_keywords: ["WIP", "DO NOT MERGE"]
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)